pattern-library/sass/utilities/_variables.scss

```diff
@@ -359,98 +359,202 @@ $input-width: (
 
 // links
-$link-color:                            palette(primary, dark) !default;
-$link-focus-color:                      palette(primary, base) !default;
-$link-visited-color:                    palette(primary, dark) !default;
+$link-color:                                palette(primary, dark) !default;
+$link-focus-color:                          palette(primary, base) !default;
+$link-visited-color:                        palette(primary, dark) !default;
+
 
 // headings
-$headings-font-weight-normal:           font-weight(normal) !default;
-$headings-font-weight-bold:             font-weight(semi-bold) !default;
-$headings-font-weight-x-bold:           font-weight(bold) !default;
+$headings-font-weight-normal:               font-weight(normal) !default;
+$headings-font-weight-bold:                 font-weight(semi-bold) !default;
+$headings-font-weight-x-bold:               font-weight(bold) !default;
+
+$headings-base-font-family:                 $font-family-sans-serif !default;
+$headings-base-font-weight:                 $headings-font-weight-normal !default;
+$headings-base-color:                       palette(grayscale, x-dark) !default;
 
-$headings-base-font-family:             $font-family-sans-serif !default;
-$headings-base-font-weight:             $headings-font-weight-normal !default;
-$headings-base-color:                   palette(grayscale, x-dark) !default;
+$headings-emphasized-font-family:           $font-family-sans-serif !default;
+$headings-emphasized-color:                 palette(grayscale, black-t) !default;
 
-$headings-emphasized-font-family:       $font-family-sans-serif !default;
-$headings-emphasized-color:             palette(grayscale, black-t) !default;
+$headings-de-emphasized-font-family:        $font-family-sans-serif !default;
+$headings-de-emphasized-color:              palette(grayscale, dark) !default;
 
-$headings-de-emphasized-font-family:    $font-family-sans-serif !default;
-$headings-de-emphasized-color:          palette(grayscale, dark) !default;
 
 // buttons
-$buttons-border-style:                  solid !default;
-$buttons-border-size:                   1px !default;
-$buttons-elevated-shadow:               3px !default;
-$buttons-border-radius:                 3px !default;
-$buttons-font-weight:                   font-weight(semi-bold) !default;
+$btn-border-style:                          solid !default;
+$btn-border-size:                           1px !default;
+$btn-shadow:                                3px !default;
+$btn-font-weight:                           font-weight(semi-bold) !default;
+$btn-border-radius:                         $component-border-radius !default;
+
+// default button
+$btn-default-border-color:                  palette(primary, base) !default;
+$btn-default-background:                    palette(grayscale, white) !default;
+$btn-default-color:                         palette(primary, base) !default;
+// $btn-default-shadow:                     ;
```

Comment:
Oh, I added these based on the button properties you and I defined within the theming inventory (https://docs.google.com/a/edx.org/spreadsheets/d/1kHttF70rf2p7-9-Ev6kubfSFwncrDi0KzwFO9jArOx0/edit?usp=sharing). 

My hope was that it was easier to sync up when we got to the platform, but if its just noise/junk, I can lose it until or if we use it.